lib/Dancer2/Template/Tiny.pm

```diff
@@ -45,25 +49,14 @@ __END__
 This template engine allows you to use L<Template::Tiny> in L<Dancer2>.
 
 L<Template::Tiny> is an implementation of a subset of L<Template::Toolkit> (the
-major parts) which takes much less memory and is faster. If you're only using
+major parts) but takes much less memory and is faster. If you're only using
 the main functions of Template::Toolkit, you could use Template::Tiny. You can
 also seamlessly move back to Template::Toolkit whenever you want.
 
-However, Dancer2 uses a modified version of L<Template::Tiny>, which is L<Dancer2::Template::Implementation::ForkedTiny>. It adds 2 features :
-
-=over
-
-=item *
-
-opening and closing tag are now configurable
-
-=item *
-
-CodeRefs are evaluated and their results is inserted in the result.
-
-=back
-
-You can read more on L<Dancer2::Template::Implementation::ForkedTiny>.
+Dancer2 maintained a fork of L<Template::Tiny> for some time that added
+several features missing in the original module. These have since been
+implemented, and as such, a minimum L<Template::Tiny> version of 1.16 is
+required.
```

Comment:
I don't think this paragraph is needed. (but then, it doesn't hurt anything, so if you want to keep it, no issue from here for that).

```suggestion
```